src/main/webapp/js/science_portal.js

```diff
@@ -118,6 +112,63 @@
 
     // ------------ Data display functions
 
+    function populateSessionList(sessionData) {
+      var $sessionListDiv = $('#spSessionList')
```

Comment:
Small gripe: HTML IDs use underscores rather than camel case.

Reply:
changed here & in the javascript